RestaurantManager/App.xaml

```diff
@@ -4,6 +4,39 @@
              xmlns:dryIoc="http://prismlibrary.com"
              x:Class="RestaurantManager.App">
     <Application.Resources>
-
+        <Color x:Key="WindowBackgroundColor">White</Color>
+        <Color x:Key="WindowBackgroundColorDark">Black</Color>
+        
+        <ResourceDictionary>
+            <OnPlatform x:TypeArguments="x:String" 
+                        x:Key="FontAwesomeBrands">
+                <On Platform="Android" 
+                    Value="FontAwesome5Brands.otf#Regular" />
+                <On Platform="iOS" 
+                    Value="FontAwesome5Brands-Regular" />
+                <On Platform="UWP" 
+                    Value="/Assets/FontAwesome5Brands.otf#Font Awesome 5 Brands" />
+            </OnPlatform> 
+    
+            <OnPlatform x:TypeArguments="x:String" 
+                        x:Key="FontAwesomeSolid"> 
+                <On Platform="Android" 
+                    Value="FontAwesome5Solid.otf#Regular" /> 
+                <On Platform="iOS" 
+                    Value="FontAwesome5Free-Solid" /> 
+                <On Platform="UWP" 
+                    Value="/Assets/FontAwesome5Solid.otf#Font Awesome 5 Free" /> 
+            </OnPlatform> 
+    
+            <OnPlatform x:TypeArguments="x:String" 
+                        x:Key="FontAwesomeRegular">
+                <On Platform="Android" 
+                    Value="FontAwesome5Regular.otf#Regular" /> 
+                <On Platform="iOS" 
+                    Value="FontAwesome5Free-Regular" /> 
+                <On Platform="UWP" 
```

Comment:
there is no need for UWP since we don't even support it

Reply:
Unnecessary fonts removed